specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2022-10-01/Clusters.json

```diff
@@ -0,0 +1,668 @@
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ClusterPatchProperties",
+          "description": "Log Analytics cluster properties."
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The identity of the resource."
+        },
+        "sku": {
+          "$ref": "#/definitions/ClusterSku",
+          "description": "The sku properties."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        }
+      },
+      "description": "The top level Log Analytics cluster resource container.",
+      "x-ms-azure-resource": true
+    },
+    "Cluster": {
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The identity of the resource."
+        },
+        "sku": {
+          "$ref": "#/definitions/ClusterSku",
+          "description": "The sku properties."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ClusterProperties",
+          "description": "Log Analytics cluster properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "The top level Log Analytics cluster resource container."
+    },
+    "ClusterListResult": {
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of recommendations.",
+          "type": "string"
+        },
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Cluster"
+          },
+          "description": "A list of Log Analytics clusters."
+        }
+      },
+      "description": "The list clusters operation response."
+    },
+    "keyVaultProperties": {
+      "type": "object",
+      "description": "The key vault properties.",
+      "properties": {
+        "keyVaultUri": {
+          "description": "The Key Vault uri which holds they key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyName": {
+          "description": "The name of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyVersion": {
+          "description": "The version of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyRsaSize": {
+          "description": "Selected key minimum required size.",
+          "type": "integer",
+          "format": "int32"
```

Comment:
are keyvault properties readonly?

Reply:
I guess not, since there's an update sample for them. You might want to include them in a create sample also then.

Reply:
They are not read-only, values are provided explicitly by customers.

Reply:
Keyvault usage is a niche, i prefer to provide the common case scenario in create example.
User assigned identity and key vault usage are covered in Update.